front/app/components/UI/QuillEditor/index.tsx

```diff
@@ -236,7 +237,8 @@ const Container = styled.div<{
     ${(props: any) => quillEditedContent(props.theme.colorMain)};
 
     .ql-editor {
-      min-height: 300px;
+      min-height: ${(props) => (props.minHeight ? props.minHeight : '300px')};
```

Comment:
I added this since we decided not to handle scroll on the Settings panel. With the toggle and delete button (and the quill editor) the Settings content is too long on small desktop screens. I added an optional min-height on the quill editor like I did for the max-height to make everything in the panel fit.

Reply:
Ok, this makes sense 👍